extensions/webview/init.lua

```diff
@@ -17,14 +17,45 @@
 ---
 --- This module provides support for injecting custom JavaScript user content into your webviews and for JavaScript to post messages back to Hammerspoon.
 
-local module       = require("hs.webview.internal")
-module.usercontent = require("hs.webview.usercontent") ;
+local USERDATA_TAG = "hs.webview"
+
+local osVersion = require"hs.host".operatingSystemVersion()
```

Comment:
I don't care enough either :)

I only pointed it out in case it was either a mistake or an unwanted style inconsistency.  Turns out, it's neither.